msteams-platform/toolkit/TeamsFx-SDK.md

```diff
@@ -50,30 +53,26 @@ npm install @microsoft/teamsfx
 
 ### Create `MicrosoftGraphClient` service
 
```

Comment:
@frankqianms Do you think this section is duplicated with the Graph Scenarios below?

Reply:
Maybe it's kind of duplicated with the Graph Scenarios, especially the code snippets. Let's disscuss it today.

Reply:
This can be done.

Reply:
This has been implemented.